autoload/fzf_preview/resource.vim

```diff
@@ -93,6 +93,15 @@ endfunction
 
 function! fzf_preview#resource#grep(args) abort
   return  fzf_preview#converter#convert_for_fzf(systemlist(fzf_preview#command#grep_command(a:args)), 1)
+
+function! fzf_preview#resource#buffer_tags() abort
+  if !filereadable(expand('%'))
+    return []
+  endif
+
+  let lines = fzf_preview#converter#convert_for_fzf(systemlist(fzf_preview#command#buffer_tags_command(expand('%'))), 1)
+  let matches = map(lines, { _, line -> matchlist(line, '^\([^\t]\+\)\t\(\S\+\)\t\(\d\+\);"') })
+  return fzf_preview#converter#convert_for_fzf(map(matches, { _, m -> m[2] . ':' . m[3] . ":\t" . m[1] }), 1)
```

Comment:
There is no point to repeat current filepath in every line. Maybe line numer is enough?
```suggestion
  return fzf_preview#converter#convert_for_fzf(map(matches, { _, m -> m[3] . ":\t" . m[1] }), 1)
```